### ISSUE TITLE
Pages giving 4xx reponse code

### ISSUE THREAD

**Issue by v-mepa** [dotnet-foundation/website#150]
Attached is the list of pages not found and are giving 4xx reponse code as per screaming frog crawler _(website audit tool)_. The list contain both internal/external links which are linked from dotnet foundation website.
4xx errors effect user experience, all the listed pages are non-indexable, so they can be either removed/deleted or can be used 301 to redirect to right place. 
If any listed pages are working and not giving 4xx, I would request you to simply ignore it. 

[Dotnet_foundation_response_codes_client_error_(4xx).xlsx](https://github.com/dotnet-foundation/website/files/4833897/Dotnet_foundation_response_codes_client_error_.4xx.xlsx)

**Comment by clairernovotny** [comment]
Most of those are false errors from the tool. It says too many redirects, but I know most of those to be valid.

**Comment by v-mepa** [comment]
I agree with you many times Screaming frog crawler does tend to show up false errors, if the rest URL's are valid and working I will close this issue, let me know :)
